js/glpi_install.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+let request_running = false;
+
+function startDatabaseInstall(message_element_id, success_html)
+{
+    const message_element = document.getElementById(message_element_id);
+    if (!message_element) {
+        const alert = document.createElement('p');
+        alert.innerText = 'Could not load HtmlElement to append messages to.';
+        document.body.appendChild(alert);
+        throw new Error(alert.innerText);
+    }
+
+    const single_message_element = document.createElement('p');
+    const msg_list_element = document.createElement('div');
+    message_element.appendChild(msg_list_element);
+    msg_list_element.appendChild(single_message_element);
+
+    function message(text) {
+        const alert = document.createElement('p');
+        alert.innerHTML = text;
+        msg_list_element.appendChild(alert);
+    }
+    function queries_message(amount) {
+        single_message_element.innerHTML = `Process: ${amount}`;
```

Comment:
Messages should be translated, but during the install process, translations are not loaded on the client side.
You should probably compute the messages to display on the PHP side of the `check_progress` endpoint.